bitnami/wavefront/README.md

```diff
@@ -0,0 +1,249 @@
+    --set wavefront.url=https://<YOUR_CLUSTER>.wavefront.com \
+    --set wavefront.token=<YOUR_API_TOKEN>
+```
+
+The command deploys Wavefront on the Kubernetes cluster in the default configuration. The [Parameters](#parameters) section lists the parameters that can be configured during installation.
+
+The **requried** parameters are `clusterName`, `wavefront.url` and `wavefront.token`. You will need to provide values for those options for a successful installation of the chart.
+
+> **Tip**: List all releases using `helm list`
+
+## Uninstalling the Chart
+
+To uninstall/delete the `my-release` deployment:
+
+```bash
+$ helm delete my-release
+```
+
+The command removes all the Kubernetes components associated with the chart and deletes the release.
+
+## Parameters
+
+The following table lists the configurable parameters of the Wavefront chart and their default values.
+
+#### Global parameters
+
+| Parameter                 | Description                                                    | Default           |
+|---------------------------|----------------------------------------------------------------|-------------------|
+| `global.imageRegistry`    | Global Docker image registry                                   | `nil`             |
+| `global.imagePullSecrets` | Global Docker registry secret names as an array                | `[]` (does not add image pull secrets to deployed pods) |
+| `global.storageClass`     | Global storage class for dynamic provisioning                  | `nil`             |
+
+#### Common parameters
+
+| Parameter                    | Description                                                 | Default                                   |
+|------------------------------|-------------------------------------------------------------|-------------------------------------------|
+| `clusterName`                | Unique name for the Kubernetes cluster (required)           | `KUBERNETES_CLUSTER_NAME`                 |
+| `wavefront.url`              | Wavefront URL for your cluster (required)                   | `https://YOUR_CLUSTER.wavefront.com`      |
+| `wavefront.token`            | Wavefront API Token (required)                              | `YOUR_API_TOKEN`                          |
+| `commonLabels`               | Labels to add to all deployed objects                       | `{}`                                      |
+| `commonAnnotations`          | Annotations to add to all deployed objects                  | `{}`                                      |
+| `extraDeploy`                | Array of extra objects to deploy with the release           | `[]` (evaluated as a template)            |
+| `containerSecurityContext`   | Container security podSecurityContext                       | `{ runAsUser: 1001, runAsNonRoot: true }` |
+| `podSecurityContext`         | Pod security                                                | `{ fsGroup: 1001 }`                       |
```

Comment:
Why are they common parameter? I should be able to set different container/pod security policies for the proxy and the collector.

Reply:
I did common following Harbor as an example where we are setting those parameters in a global way instead of per deployment/container, but I am changing it